torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -413,6 +416,34 @@ Operation listAdd(Node* node) {
   };
 }
 
+template <typename TList, typename TElement>
+Operation listSlice(Node* node) {
+  return [=](Stack& stack) {
```

Comment:
You're not using any thing from the surrounding scope, so why have a capture at all? I think `[]` will be better. If someone adds a parameter later on it will get silently copied into the lambda closure